### PR TITLE
add new test suite

### PR DESCRIPTION
**Pull request by byashimov**
- Adds new test suite
- Adds tests for the following service integrations: clickhouse_postgresql, kafka_logs, kafka_connect
- Adds a new action and renames an old make command: test -> test-ginko, new -> test

### CODE REVIEW

**Comment by byashimov** [comment]
tests https://github.com/aiven/aiven-operator/actions/runs/4280889726

**Review by ivan-savciuc — COMMENTED** [review]
I'm testing it locally and getting strange results:
1. It runs really fast, so the test reports that it is done while services are not yet `RUNNING` on Aiven. How this can be so fast?
image
2. I have lots of panic's in the logs during runtime:
```
 unable to create or update aiven secret: Object default/kafka-secret is already owned by another Kafka controller test-k8s-0wqtydi-ks"}
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
        /home/ivan.savciuc/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.2/pkg/internal/controller/controller.go:273
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
        /home/ivan.savciuc/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.2/pkg/internal/controller/controller.go:234
```
3. And lots of errors on the start of the test:
```
1.6776915844250357e+09	ERROR	Reconciler error	{"controller": "kafka", "controllerGroup": "aiven.io", "controllerKind": "Kafka", "kafka": {"name":"test-k8s-xvqor1e-ks","namespace":"default"}, "namespace": "default", "name": "test-k8s-xvqor1e-ks", "reconcileID": "f2ff50bd-edbc-42cb-a435-b2f99e9f225e", "error": "unable to wait until instance is running: unable to create or update aiven secret: Object default/kafka-secret is already owned by another Kafka controller test-k8s-meteggi-ks"}
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
	/home/ivan.savciuc/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.2/pkg/internal/controller/controller.go:273
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
	/home/ivan.savciuc/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.2/pkg/internal/controller/controller.go:234
```

So all in all I have tests reporting: `--- PASS: TestServiceIntegrationKafkaLogs (237.99s)` after 3 min, I have 146 errors in the logs.

File: main.go [hunk 1]
```diff
@@ -439,8 +176,6 @@ func main() {
 		}
 	}
 
-	//+kubebuilder:scaffold:builder
```
[the remark this answers is not in the document]
Comment (byashimov) [reply main.go:442]: It would add it in a standard way, and you would have to rewrite it entirely.

**Comment by byashimov** [comment]
> I'm testing it locally and getting strange results:
> 
> 1. It runs really fast, so the test reports that it is done while services are not yet `RUNNING` on Aiven. How this can be so fast?
>    image
> 2. I have lots of panic's in the logs during runtime:
> 
> ```
>  unable to create or update aiven secret: Object default/kafka-secret is already owned by another Kafka controller test-k8s-0wqtydi-ks"}
> sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
>         /home/ivan.savciuc/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.2/pkg/internal/controller/controller.go:273
> sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
>         /home/ivan.savciuc/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.2/pkg/internal/controller/controller.go:234
> ```
> 
> 3. And lots of errors on the start of the test:
> 
> ```
> 1.6776915844250357e+09	ERROR	Reconciler error	{"controller": "kafka", "controllerGroup": "aiven.io", "controllerKind": "Kafka", "kafka": {"name":"test-k8s-xvqor1e-ks","namespace":"default"}, "namespace": "default", "name": "test-k8s-xvqor1e-ks", "reconcileID": "f2ff50bd-edbc-42cb-a435-b2f99e9f225e", "error": "unable to wait until instance is running: unable to create or update aiven secret: Object default/kafka-secret is already owned by another Kafka controller test-k8s-meteggi-ks"}
> sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
> 	/home/ivan.savciuc/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.2/pkg/internal/controller/controller.go:273
> sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
> 	/home/ivan.savciuc/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.12.2/pkg/internal/controller/controller.go:234
> ```
> 
> So all in all I have tests reporting: `--- PASS: TestServiceIntegrationKafkaLogs (237.99s)` after 3 min, I have 146 errors in the logs.

I have updated PR. There were racing secrets. Nice to know. I wonder if that racing right now in ginkgo. As to the timing: I have no idea what is going on. Right now I'm running the tests and it takes 236.730s.

**Review by ivan-savciuc — APPROVED** [review]
Works now :+1:

**Pull request merged**